recipes/vegafusion-python-embed/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "vegafusion-python-embed" %}
+{% set name_under = "vegafusion_python_embed" %}
+{% set version = "0.0.4_rc.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name_under[0] }}/{{ name_under }}/{{ name_under }}-{{ version }}.tar.gz
+  sha256: d5582e82ec482e9327ab53ac6047826effa0e4ac6bc1d0e81b9fe5818f09d1da
+
+build:
+  number: 0
+  script: |
+    export RUSTFLAGS="--cfg unsound_local_offset"     # [not win]
+    set RUSTFLAGS=--cfg unsound_local_offset          # [win]
```

Comment:
I think the "safe way" is to put a quotes around the entire env var plus value, like `set "RUSTFLAGS=--cfg unsound_local_offset"`

Reply:
Thanks! Giving that a try now